### PR TITLE
Text KPI chart added

### PR DESCRIPTION
**Pull request by hjkmarshall**
Title and description bottom-margin reduced
Additional consts added for font-sizes and colors
New Text KPI chart added
Download turned off as default for kpi charts
KPI charts renamed to mention number vs text

### CODE REVIEW

File: src/components/vanilla/charts/KPIChart/index.tsx [hunk 1]
```diff
@@ -12,82 +12,92 @@ type Props = {
...
       n: formatValue(n, {
         type: 'number',
-        meta: props.metric?.meta,
-        dps: props.dps
+        meta: metric?.meta,
+        dps: dps
       })
     };
-  }, [props]);
+  }, [results, prevResults, metric, dps, dimension]);
```
Comment (mad-raccoon) [inline src/components/vanilla/charts/KPIChart/index.tsx:41]: image

Comment (mad-raccoon) [reply src/components/vanilla/charts/KPIChart/index.tsx:41]: can we have this dependency?

**Review by mad-raccoon — CHANGES_REQUESTED** [review]

File: src/components/vanilla/charts/KPIChart/index.tsx (hunk 1 above)
Comment (hjkmarshall) [reply src/components/vanilla/charts/KPIChart/index.tsx:41]: @mad-raccoon do you add props.results.data.length as a depedency?

Comment (mad-raccoon) [reply src/components/vanilla/charts/KPIChart/index.tsx:41]: yes !

Comment (mad-raccoon) [reply src/components/vanilla/charts/KPIChart/index.tsx:41]: [results, prevResults, metric, dps, dimension, props.results.data.length]

**Review by mad-raccoon — APPROVED** [review]

**Pull request merged**